src/styles/card/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+  }
+
+  &-container {
```

Comment:
The name `stars-container` does not align with BEM methodology. The `container` should be an element of `stars`, not a modifier. Consider renaming to `stars__container`.